helper/impls/helper-impl-17/src/main/java/com/linkedin/avroutil1/compatibility/avro17/FieldBuilder17.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility.avro17;
+
+import com.linkedin.avroutil1.compatibility.FieldBuilder;
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Field.Order;
+import org.codehaus.jackson.JsonNode;
+
+
+public class FieldBuilder17 implements FieldBuilder {
+  private static Schema.Field _field;
```

Comment:
why is this field static? also i think the Schema.Field (original) should be final and set at constructor time for the builder

Reply:
yes it should be final but not static.